### ISSUE TITLE
Nightly Debian packages using incorrect installation paths

### ISSUE THREAD

**Issue by mc-butler** [MidnightCommander/mc#4168]
> [!IMPORTANT]
> This issue was migrated from Trac:
>
> | **Origin** | https://midnight-commander.org/ticket/4168 |
> | :------------- | :------------------------------------------------------------------------------------------------ |
> | **Reporter** | `RogueScholar` (admin@….net) |
> | **Mentions** | `IceMan` (@tataranovich) |
> | **Keywords** | `nightly`, `Debian`, `tataranovich`, `binary`, `deb` |

<p>
First a long-overdue expression of my ongoing gratitude to @<tt>IceMan</tt> for maintaining and operating the infrastructure that provides these development builds to those of us running Debian-based distros. Efforts such as those are among the most thankless and unglamorous in the open-source ecosystem and nothing I'm calling attention to in this ticket is meant as criticism.<br />
</p>
<p>
While not impacting a large number of files in the built packages, those that are do happen to be somewhat significant: the shell snippets that define the aliases for the mc invocation. They are being installed to /usr/share/mc/bin rather than the intended destination of /usr/lib/mc. (Wouldn't the FHS 3.0 prefer they be lodged in /usr/libexec/mc instead?) It's probably also worth noting that the mc.sh file is rendered non-functional in the process since its generated by the build system and still defines the alias for mc with a reference to /usr/lib/mc/mc-wrapper.sh despite the file not being installed there. Oh, and the pixmap file for mcedit (mcedit.xpm) is absent altogether, yet the ancient Debian menu file is provided though even Debian doesn't ship it anymore.<br />
</p>
<p>
Basically, it seems like a sync with the <a class="ext-link" href="https://salsa.debian.org/debian/mc">Debian packaging repository</a> is probably in order to iron this out.<br />
</p>
<p>
I hope you're all doing well and staying healthy and in store for a prosperous new year.<br />
</p>
